auxiliaries/pipeline_auxiliaries.py

```diff
@@ -521,11 +528,51 @@ def log_scale(df, rank_method):
     return df
 
 
-def change_key_name(old_names_dict, map_name):
+def change_key_name(old_names_dict, map_name, reverse = False):
     new_dict = {}
     if old_names_dict:
         keys = old_names_dict.keys()
         for key in keys:
             new_dict[map_name[key]] = old_names_dict[key]
         return new_dict
     return old_names_dict
+
+
+def process_params(args, config_path, map_name_parameters, func_run, phase, argv):
+    # create data structure for running filter_reads
+    done_file = args.done_file_path
+    base_map =  args.__dict__
+    keys = base_map.keys()
+    base_map = change_key_name(base_map, map_name_parameters)
+    if config_path:    
+        f = open(config_path)
+        multi_experiments_dict = json.load(f)
+        # validation of the json file
+        is_valid = is_valid_json_structure(config_path, multi_experiments_dict, schema_to_phase[phase], logger)
+        if not is_valid:
+            return 
+        configuration = multi_experiments_dict['configuration']
+        base_map.update(configuration)
+        runs = multi_experiments_dict['runs']
+        for run in runs:
+            dict_params = base_map.copy()
+            if phase == 'model_fitting':
+                dict_params.update(runs[run]['configuration'])
+            else:
+                dict_params.update(runs[run])
+            # create new list of argv of the specific run.
+            f = lambda k: k
+            g = lambda k: str(dict_params[map_name_parameters[k]])

```

Comment:
code should have meaningful names, we are not restricted to mathematical notations 😃.
`get_key` and `get_value` or something similar are clearer